test/models/test_albef.py

```diff
@@ -4,113 +4,221 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
+from functools import partial
+
+import pytest
 import torch
 from test.test_utils import assert_expected, set_rng_seed
 from torch import nn, Tensor
 from torchmultimodal.models.albef import ALBEFModel, ALBEFSimilarity
+from torchmultimodal.modules.encoders.albef_multimodal_encoder import (
+    ALBEFMultimodalEncoder,
+)
+from torchmultimodal.modules.encoders.albef_text_encoder import ALBEFTextEncoder
+from torchmultimodal.modules.encoders.albef_vision_encoder import ALBEFVisionEncoder
+
+
+@pytest.fixture(autouse=True)
+def vision_encoder():
+    set_rng_seed(0)
```

Comment:
I see, in that case it is probably fine to just leave the `set_rng_seed` inside vision_encoder for now. I am not sure how it work, but I guess probably because it is the first function or the first one being called.